site/docs/guides/saving-data.md

```diff
@@ -37,6 +37,11 @@ var gameData int32 = 1337
 w4.DiskW(unsafe.Pointer(&gameData), unsafe.Sizeof(gameData))
 ```
 
+```nim
+var gameData = 1337'i32
+discard diskw(addr gameData, uint32 sizeof(gameData))
```

Comment:
This is needed because Nim doesn't "throw away" results of functions if you don't use them - instead, you have to explicitly say `discard` to signify that you don't need the result.

Reply:
Makes sense, thanks for the context!

Is the `uint32` part of `uint32 sizeof(gameData)` a cast? Would it make sense to change the type of that param in `diskw/diskr` to make the cast unnecessary?

Reply:
@aduros it's a type conversion which is safe (Nim does also have casts which look different). I can change the type and it'll still work because uint32 and Nim's `int` will both be 32-bit on wasm32, but on any other platform it'll be potentially incorrect

Reply:
So what would be the way to go here?

Reply:
Whatever you decide sounds good to me. I think we can always change it later if we really wanted to without breaking existing Nim projects.